File: JAVA-Basic/JavaBasic/src/ConditionalStatementsAdvanced/Exercise/NewHouse.java
package ConditionalStatementsAdvanced.Exercise;

import java.util.Scanner;

public class NewHouse {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String flowers = scan.nextLine();
        int flowersNum = Integer.parseInt(scan.nextLine());
        int budget = Integer.parseInt(scan.nextLine());
        double price = 0.0;

        switch (flowers) {
            case "Roses":
                price = flowersNum * 5;
                if (flowersNum > 80) {
                    price *= 0.9;
                }
                break;
            case "Dahlias":
                price = flowersNum * 3.8;
                if (flowersNum > 90) {
                    price *= 0.85;
                }
                break;
            case "Tulips":
                price = flowersNum * 2.8;
                if (flowersNum > 80) {
                    price *= 0.85;
                }
                break;
            case "Narcissus":
                price = flowersNum * 3;
                if (flowersNum < 120) {
                    price *= 1.15;
                }
                break;
            case "Gladiolus":
                price = flowersNum * 2.5;
                if (flowersNum < 80) {
                    price *= 1.2;
                }
                break;
        }
        double priceDifference = Math.abs(price - budget);
        if (budget >= price) {
            System.out.printf("Hey, you have a great garden with %d %s and %.2f leva left.", flowersNum, flowers, priceDifference);
        } else {
            System.out.printf("Not enough money, you need %.2f leva more.", priceDifference);
        }
    }
}

File: 01.JS-Basic/По сложни проверки Упражнение/NewHouse.js
function newHouse(input) {

    let flowerType = input[0]
    let flowerNum = Number(input[1])
    let budget = Number(input[2])

    let price = 0.0

    switch (flowerType) {
        case "Roses": if (flowerNum > 80) {
            price = 5 * 0.9
        }
        else { price = 5 }; break;

        case "Dahlias": if (flowerNum > 90) {
            price = 3.8 * 0.85
        }
        else { price = 3.8 }; break;

        case "Tulips": if (flowerNum > 80) {
            price = 2.8 * 0.85
        }
        else { price = 2.8 }; break;

        case "Narcissus": if (flowerNum < 120) {
            price = 2.8 * 1.15
        }
        else { price = 3 }; break;

        case "Gladiolus": if (flowerNum < 80) {
            price = 2.5 * 1.2
        }
        else { price = 2.5 }; break;
    }
    let totalPrice = price * flowerNum
    if (budget >= totalPrice) {
        console.log(`Hey, you have a great garden with ${flowerNum} ${flowerType} and ${(budget - totalPrice).toFixed(2)} leva left.`)
    }
    else { console.log(`Not enough money, you need ${(totalPrice - budget).toFixed(2)} leva more.`) }

}
newHouse(["Tulips",
"88",
"260"])

